### PR TITLE
Increase disease events spawn chance

### PR DESCRIPTION
**Pull request by JJRcop**
:wink: [​]

[​]: https://github.com/bradfitz/gitbrute

### CODE REVIEW

**Comment by Shadowflame909** [comment]
You gotta buff sentient disease first Plague Doctor

**Comment by necromanceranne** [comment]
Bad timing op

File: code/modules/antagonists/disease/disease_event.dm [hunk 1]
```diff
@@ -2,8 +2,8 @@
 /datum/round_event_control/sentient_disease
 	name = "Spawn Sentient Disease"
 	typepath = /datum/round_event/ghost_role/sentient_disease
-	weight = 7
-	max_occurrences = 1
+	weight = 25
+	max_occurrences = 2
```
Comment (ATH1909) [inline code/modules/antagonists/disease/disease_event.dm]: 👀

File: code/modules/events/disease_outbreak.dm [hunk 2]
```diff
@@ -1,9 +1,9 @@
 /datum/round_event_control/disease_outbreak
 	name = "Disease Outbreak"
 	typepath = /datum/round_event/disease_outbreak
-	max_occurrences = 1
+	max_occurrences = 6
```
Comment (ATH1909) [inline code/modules/events/disease_outbreak.dm]: 👀

**Comment by Tlaltecuhtli** [comment]
the random diseases are a joke as most of the time you wont notice them between the ones created by miasma, sentient aids isnt that fun or interesting (uh oh the "regen coma pls dont heal " virus  has 2 cures.... what could it mean) for 4x more chance of happening

**Comment by HomocidlePancakes** [comment]
I don’t even remember the last time I saw a unique virus like the Spanish Flu or GBS. This is needed tbh. Also, their cures are pretty easy to obtain, so somebody should change that too.

**Comment by XDTM** [comment]
Maybe we could let ghosts vote on the symptom selection

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Pull request closed without merging**